data-raw/healthy-lives/behavioural-risk-factors/alcohol-misuse.R

```diff
@@ -0,0 +1,87 @@
+# Load packages
+library(tidyverse)
+library(readxl)
+library(sf)
+library(httr)
+library(httr)
+library(dplyr)
+
+# URL of the Excel file
+excel_url <- "https://www.ons.gov.uk/file?uri=/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/datasets/alcoholspecificdeathsinenglandandwalesbylocalauthority/current/alcoholspecificdeathsbylocalauthority2022.xlsx"
+
+# Download the file to a temporary location
+temp_file <- tempfile(fileext = ".xlsx")
+download.file(excel_url, temp_file, mode = "wb")
+
+# Read the Excel file into R
+excel_data <- readxl::read_excel(temp_file)
+
+# Print a summary or inspect the data
```

Comment:
Print statements are required for you to inspect the data as you are developing the script, it is not required in the processing of the script and therefore should be removed.

Please go through your script and remove any lines used to inspect or check the data.